README.adoc

```diff
@@ -342,27 +342,39 @@ Name the models with meaningful (but short) names without abbreviations.
 
 === ActiveAttr Gem [[activeattr-gem]]
 
-If you need model objects that support Active Record behavior (like validation) without the Active Record database functionality use the https://github.com/cgriego/active_attr[ActiveAttr] gem.
+If you need objects that support ActiveRecord-like behavior (like validations) without the database functionality, use `ActiveModel::Model`.
```

Comment:
I think you should also update the title here.

Reply:
You are correct. Fixed.